content/curriculum.md

```diff
@@ -22,44 +19,83 @@
 
 #### Goals
 
-Answer the following questions:
-
-* What determines the quality of a dataset?
-* What are common data formats?
-* What are common metadata standards?
-* How can I fix "bad" data?
-* What can I do if my data still doesn't look right? (e.g. it's possible nobody living
-  knows how to resolve the problem, metadata may be incorrectly documented and you may
-  need help from the data producer, briefly mention techniques like georegistration,
-  bias correction)
+* Get to know each other
+* Get to know tools you'll need for this workshop
```

Comment:
Lets consider avoiding contractions (`you'll`, `we'll`, etc). This might improve readability for those who know English as a second language.